### DISCUSSION TITLE
LCD YTC320124LAC-035-100N with Tasmota

### DISCUSSION THREAD

**Discussion by M4rkoV2** [arendst/Tasmota#21938] [Q&A]
Hi,

Could this LCD: https://www.andersdx.com/media/downloads/displays/colour-displays/tft-displays/YTC320124LAC-035-100N.pdf  work with Tasmota on ESP32-S3?

**Comment by Jason2866** [comment]
There is currently no display support for devices using the ST7272A controller. If the display controller is similar or compatible to the currently known and working ST7262, ST7735, ST7789 controllers there is a chance to get it working with Tasmota. This is not an quick and easy job for beginners.
